### DISCUSSION TITLE
Projet

### DISCUSSION THREAD

**Discussion by Aliou9796** [community/community#135816] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Je veux créer une application de gestion de véhicules en location mais je n'ai aucune expérience j'ai besoin d'aide

**Comment by sojanonelson** [comment]
Hi @Aliou9796 
It's great that you want to build a vehicle rental management app! Here's how to get started:

1. Plan:
List essential features (vehicle management, bookings, customer profiles, payments).
Sketch out how users will interact with the app.
2. Choose a Tech Stack:
For web: Use HTML/CSS/JavaScript or frameworks like React.js.
For mobile: Try React Native or Flutter.
Use a backend like Node.js or Django and a database like MySQL or MongoDB.
3. Learn the Basics:
Use free resources like MDN Web Docs, freeCodeCamp, or YouTube to understand web/mobile development.
4. Start Small:
Build a simple version (e.g., adding and listing vehicles) and expand from there.
5. Use Libraries:
Leverage existing libraries (e.g., Mongoose for databases, Stripe for payments) to speed up development.
Keep learning and join coding communities for help when needed!

Good luck!

**Comment by Aliou9796** [comment]
Merci est-ce que je peux avoir un modèle déjà conçu comme référence

**Comment by ebndev** [comment]
Hi @Aliou9796, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
